.gitlab-ci.yml

```diff
@@ -354,3 +355,16 @@ coverage:
       - compile-xop-debug-coverage-macosx
       - download-xoptoolkit
       - testing-macosx
+
+symbol_creation_and_upload:
+  tags:
+    - debian, docker
+  stage: fifth
+  image:
+    name: ${CI_REGISTRY}/internal/docker-utils:latest
+  needs:
+    - compile-libzmq-windows
```

Comment:
@MichaelHuth Why that dependency and not https://github.com/AllenInstitute/ZeroMQ-XOP/blob/4c5928d22928f51f30fdcc24e600c59592e7cd5b/.gitlab-ci.yml#L111? And shouldn't we restrict it to main?

Reply:
Ok I fixed this.